src/views/content/content.scss

```diff
@@ -14,6 +14,16 @@
     }
 }
 
+.content-header {
+    font-size: 28px;
```

Comment:
Maybe better would be to create a `@mixin text-style-title-l` for this in `fonts.scss`

Reply:
Introduced and used `text-style-title-l` mixin